### PR TITLE
Plane: Add airbrake documentation.

### PR DESCRIPTION
**Pull request by samuelctabor**
Fairly brief right now.

### CODE REVIEW

**Comment by Hwurzburg** [comment]
@samuelctabor Sam, as I was going thru this, I realized that the basic reverse thrust stuff was incomplete and confusing....so I redid that section extensively. Then was faced as to how to interact with your stuff....so I just worked from that base....
several things I did besides updating thrust reversing in general:
-restructured some links outside of the airbrakes-on-plane page to fit into an upcoming features scenario
-made an assumption on how to setup airbrakes on non-reversed esc setups...and that airbrake automation followed that of reverse thrust (ie works in throttle controlled modes in general via USE_REV_THRUST bitmask)...if my assumption is wrong, we need to discuss

File: plane/source/docs/airbrakes-on-plane.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Assign a spare RC channel on your transmitter to a dial, slider or 3 position switch. On the flight controller, ensure this channel is calibrated and assign this channel the RC channel ``RCx_OPTION`` option Airbrake (210).
+
+.. _airbrake-setup:
+
+
+Automatic Airbrake Setup
+========================
+
+You can have airbrakes automatically used in an automated landing or in conjunction with Reverse Thrust operation during certain throttled controlled modes or automatic landings and throttle-controlled descents, such as CRUISE mode while soaring. If manual control has been enabled using the ``RCX_OPTION`` (64), then the airbrake deployment will be the greater of either the automatic demand or manual control.
```
Comment (samuelctabor) [inline plane/source/docs/airbrakes-on-plane.rst:34]: OR if RCX_OPTION isn't configured and the USE_REV_THRUST bitmask allows it

Comment (samuelctabor) [reply plane/source/docs/airbrakes-on-plane.rst:34]: Perhaps the part in automatic landing that explains when reverse thrust is available could be make into a section and linked from here?

File: plane/source/docs/airbrakes-on-plane.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+--------------------------------
+
+Setup :ref:`Reverse Thrust<reverse-thrust>` as desired. Airbrake operation will be in sync with reverse throttle applied.
+
+Use Without Reverse Thrust ESC
+------------------------------
+
+Same as above, however:
+
+- Set :ref:`SERVO3_TRIM<SERVO3_TRIM>` = :ref:`SERVO3_MIN<SERVO3_MIN>` assuming SERVO 3 output is the throttle. This will assure that low stick and minimum demanded throttle is always SERVOx_MIN.
```
Comment (samuelctabor) [inline plane/source/docs/airbrakes-on-plane.rst:46]: Correct

**Comment by samuelctabor** [comment]
Hi @Hwurzburg all the info is correct. However there is now only incidental mention of airbrakes on the automatic landing page - originally I was trying to show that they are 100% equivalent to ESC reverse thrust. Happy to add that info back in on top of your changes if you're happy with that structure?

**Comment by Hwurzburg** [comment]
sure...but add it under the note on the future wiki changes page for auto landing....feel free to add the first comment to the airbrakes page.....and you can add the link on the auto landing page and refer to it on the airbrakes page...then squash everything for merge....thanks

**Comment by samuelctabor** [comment]
@Hwurzburg I assumed this PR would be delayed merging until the airbrake feature is in a release? Maybe I misunderstood how you're handling new features.

**Comment by Hwurzburg** [comment]
@samuelctabor here is how I try to handle it:
there are two pages: the Upcoming Features page shows in the left sidebar of each vehicle's wiki and contains the TOC entries for  new feature complete pages, like the airbrakes-on-plane........for little things, there is also a link on that page to "other wiki changes" for things that are just a paragraph on an existing page....that way, we dont have a bunch of "only available in firmware versions later than XX.XX" all thru the wiki....because, eventually they have to be hunted down and eliminated after two releases...this way, everything in master only, are found via the Upcoming Features page....when and if they integrated into a beta, then they get moved onto the wiki page, and complete pages get added to the appropriate TOC landing page....and removed from Upcoming features....
I also keep a list of master only stuff in the issues list to not lose them if docs are pending: https://github.com/ArduPilot/ardupilot_wiki/issues/1756

**Comment by samuelctabor** [comment]
@Hwurzburg okay good now I understand. In that case I'll make some minor updates to this PR and open a parallel PR for merging once the feature is in a release version.

**Comment by Hwurzburg** [comment]
@samuelctabor Unless there are other changes on this, can you squash so I can merge?feel free to do another PR, but its really not necessary....since these are just cut and paste from upcoming features/wiki changes pages when they do appear in beta releases...I set this up so all that is easy to do when the time comes...

**Comment by samuelctabor** [comment]
A couple of minor changes to make, I should get to it today.

**Comment by samuelctabor** [comment]
Squashed, ready for merge

**Comment by Hwurzburg** [comment]
merging thanks

**Pull request merged**